### ISSUE TITLE
Basic navigation in input field not working

### ISSUE THREAD

**Issue by blindahl** [continuedev/continue#2771]
### Before submitting your bug report

- [X] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [X] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [X] I've seen the [troubleshooting guide](https://docs.continue.dev/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS: Windows 11
- Continue version: 0.0.80 EAP
- IDE version: 
IntelliJ IDEA 2024.2.4 (Ultimate Edition)
Build #IU-242.23726.103, built on October 23, 2024
```

### Description

Navigating using basic keyboard input like ctrl+arrow keys isn't working. Nothing happens. When using only arrow keys the navigation works.
Also when doing Ctrl+backspace it should (at least in Windows) delete the word to the left but it removes all text entered in the current input field.

### To reproduce

1. Select som text
2. Press Ctrl+J
3. Try to navigate given Ctrl+arrow keys or use Ctrl+Backspace to try to remove only the word to the left.

### Log output

```Shell
Nothing in the log of interest.
```

**Comment by granstel** [comment]
Same with Ctrl+C/Ctrl+V in input field, and copying doesn't work at the answer area

**Comment by blindahl** [comment]
> I have the same with Ctrl+C/Ctrl+V in input field

Yeah that's really frustrating as well that copy-paste isn't working. I wasn't certain that this issue isn't reported already so didn't mention it in this report.

Overall it's annoying that the Continue window isn't conforming to how tool windows should behave in IntelliJ like escape should release  focus but I'll report that in another ticket if it's not already reported.

**Comment by Patrick-Erichsen** [comment]
Hi all, we're recommending folks downgrade to 0.0.75 at the moment. Please see this main issue for details: https://github.com/continuedev/continue/issues/2956
